components/Pagination/Pagination.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { number } from 'prop-types';
+import LeftAngleIcon from 'static/images/icons/FontAwesome/angle-left-solid.svg';
+import RightAngleIcon from 'static/images/icons/FontAwesome/angle-right-solid.svg';
+import PaginationItem from './PaginationItem/PaginationItem';
+import styles from './Pagination.css';
+
+Pagination.propTypes = {
+  currentPage: number.isRequired,
+  totalPages: number.isRequired,
+};
+
+const developmentErrors = {
```

Comment:
If you export this, you can import it into the test file and reference it when making assertions about the errors. That way, you don't need to hard-code strings multiple times!

Reply:
Nice. I have pushed with these changes.